src/main/java/racingcar/Game.java

```diff
@@ -0,0 +1,57 @@
+package racingcar;
+
+import java.util.List;
+import java.util.Random;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Game {
+	private static final String WINNER_NAME_DELIMITER = ", ";
+	private static final String NEGATIVE_ERROR_MESSAGE = "[ERROR] 음수를 입력할 수 없습니다";
+	private static final int ERROR_CRITERIA_VALUE_ZERO = 0;
+
+	private final Random random = new Random();
+
+	public void start() {
+		Cars cars = new Cars(CarFactory.of(InputView.inputCarNames()));
+		Set<Car> carSet = cars.getCars();
+		int count = validateGameCount(InputView.inputGameCount());
+		OutputView.printGameResultTitle();
+		for (int i = 0; i < count ; i++) {
+			play(carSet);
+			showResult(carSet);
+		}
+		showWinner(Referee.judgeWinner(carSet, cars.getMaxPosition()));
+	}
+
+	public void play(Set<Car> cars) {
+		for (Car car : cars) {
+			boolean canCarMove = Referee.canCarMove(makeRandomValue());
+			car.move(canCarMove);
+		}
+	}
+
+	public void showResult(Set<Car> cars) {
+		for (Car car : cars) {
+			OutputView.printLineString(car.toString());
+		}
+		OutputView.printBlankLine();
+	}
+
+	public void showWinner(List<Car> winnerCars) {
+		String winnerNames = winnerCars.stream()
+			.map(Car::getName).collect(Collectors.joining(WINNER_NAME_DELIMITER));
```

Comment:
```suggestion
		String winnerNames = winnerCars.stream()
			.map(Car::getName)
			.collect(Collectors.joining(WINNER_NAME_DELIMITER));
```
메서드 체이닝은 한줄에 한개씩 표현해주는 것이 명확하답니다~

Reply:
넵! 한줄에 한개씩 표현해주는 것, 한단계씩 처리해주는 것! 잊지 않겠습니다!